test/unit/streaming-client.spec.ts

```diff
@@ -38,12 +38,14 @@ describe('streaming-client', () => {
                 `?access_token=${token}` +
                 `&content_type=${audioConfig.getContentTypeString()}` +
                 `&user_agent=${encodeURIComponent(`RevAi-NodeSDK/${sdkVersion}`)}` +
-                `&metadata=${encodeURIComponent(config.metadata)}`
+                `&metadata=${encodeURIComponent(config.metadata)}` +
+                `&custom_vocabulary_id=${encodeURIComponent(config.customVocabularyID)}` +
+                `&filter_profanity=${encodeURIComponent(config.filterProfanity)}`
             );
             expect(mockClient.connect).toBeCalledTimes(1);
         });
 
-        it ('does not add metadata if no config provided', () => {
+        it ('does not add optional parameters if no config provided', () => {
```

Comment:
I think we also need a test for if each optional parameter is null/ not null

Reply:
The not null case is covered by the test in which all configs are provided. I can add tests for the null case however.

Reply:
Addressed.